packages/postcss/postcss.js

```diff
@@ -24,14 +24,18 @@ module.exports = postcss.plugin("modular-css", (opts) =>
                 return processor.output();
             })
             .then((output) => {
-                var { json } = processor.options;
-                
+                var { json, to } = processor.options;
```

Comment:
I'm not sure this is the `to` we should be using. It's the same as the one from `result.opts` due to how the processor just greedily copies all the available options, but I think it'd be clearer to use the `result.opts.to` value instead.

That way if the `processor` ever goes crazy and starts changing `to` this plugin will still work w/ the correct value direct from `postcss`

Reply:
Okay, I'll update that.